assets/styl/_helpers/_block.styl

```diff
@@ -1,43 +1,99 @@
 // Helpers: Block elements
 //
-// Some default classes are available for block elements:
+// Block elements helpers.
+//
+// Weight: 12
+//
+// Styleguide Helpers.BlockElements
+
+// Floating blocks
 //
 // - **.float-left**:   float left
 // - **.float-right**:  float right
+// - **.clear**:        clear both
 // - **.clearfix**:     clear floats (must be applied on the parent container)
+//
+// **Usage examples**
+// ```
+// /* example 1 */
+// <div class="clearfix">
+//   <p class="float-right">...</p>
+//   <p class="float-left">...</p>
+//   /* .clearfix class on parent div will break paragraphs' float */
+// </div>
+//
+// /* example 2 */
+// <p class="float-right">...</p>
+// <p class="float-left">...</p>
+// <p class="clear">/* .clear class will break paragraphs' floar */</div>
+// ```
+//
+// Styleguide Helpers.BlockElements.Floats
+.float-left
+    float left!important // @stylint ignore
+
+.float-right
+    float right!important // @stylint ignore
+
+.clear
+    clear both
+
+.clearfix
+    {clearfix}
+
+// Inline blocks
+//
 // - **.inline**:       display inline of a block or a list
 // - **.inline-block**: display inline-block
-// - **.hidden**:       display none
-// - **.vsb-hidden**:   visibility hidden
 //
-// Weight: 12
+// **Usage example**
+// ```
+// <ul class="inline">
+//   /* Applied on a list, .inline class will make all list-item display inline */
+//   <li>...</li>
+//   <li>...</li>
+// </ul>
 //
-// Markup: helpers-block.twig
+// <p class="inline-block">...</p>
+// ```
 //
-// Styleguide Helpers.BlockElements
+// Styleguide Helpers.BlockElements.Inline
 .inline
     {inlined}
 
 .inline-block
     display inline-block
 
+// Hidden blocks
+//
+// - **.hidden**:       display none
+// - **.vsb-hidden**:   visibility hidden
+//
+// **Usage example**
+// ```
+// <p class="hidden">...</p>
+// <p class="vsb-hidden">...</p>
+// ```
+//
+// Styleguide Helpers.BlockElements.Hide
 .hidden
     display none
 
 .vsb-hidden
     visibility hidden
 
-.float-left
-    float left!important // @stylint ignore
-
-.float-right
-    float right!important // @stylint ignore
-
-.clear
-    clear both
-
-.clearfix
-    {clearfix}
-
+// Full width blocks
+//
+// - **.responsive**:   makes an element stretch full width of its parent container
+//
+// **Usage example**
+// ```
+// <div>
+//   <img class="responsive" src="..." />
+//   /* image will be full width, with height auto */
+// </div>
+// ```
+//
+// Styleguide Helpers.BlockElements.FullWidth
 .responsive
```

Comment:
Yes. I prefer `.fullwidth` because dash is use to identify component children ans modifiers.

but `.full-width` can be better if we have a `.full-height` helper. collection off helpers `.full` like collection `.float`